lib/web_ui/lib/src/engine/platform_views/content_manager.dart

```diff
@@ -10,6 +10,10 @@ import '../embedder.dart';
 import '../util.dart';
 import 'slots.dart';
 
+// Keep these values in sync with the ones used in the framework's `HtmlElementView`.
+const String _kDefaultVisibleViewType = '_default_document_create_element_visible';
+const String _kDefaultInvisibleViewType = '_default_document_create_element_invisible';
```

Comment:
Should these two consts be made public through the `PlatformViewRegistry` in package:ui_web, so the framework doesn't need to rely on having a copy of them in sync?

Reply:
Great idea!